polyglotdb/corpus/summarized.py

```diff
@@ -498,20 +472,27 @@ def encode_measure(self, measure):
 
         """
         res = None
-        data_type = 'word'
+        speaker = False
```

Comment:
I'm not a huge fan of this style of API.  A better one would be to have encode_measure have a signature like:

```
encode_measure(self, data_name, statistic, annotation_type, by_speaker = False)
```
Where data_name would be the aspect to summarize (duration, pitch, formants, etc, some fixed number of levels that are easily expandable), statistic would be how to summarize (mean, median, sd, etc), annotation type is straightforward, and then by_speaker would be a flag for by-speaker annotations.

Essentially, the single argument of measure has 3 subparts to it, which should be split up into independent parameters